receiver/filereceiver/README.md

```diff
@@ -0,0 +1,27 @@
+This receiver will read pipeline data from JSON files. The data is written in
+[Protobuf JSON
+encoding](https://developers.google.com/protocol-buffers/docs/proto3#json)
+using [OpenTelemetry
+protocol](https://github.com/open-telemetry/opentelemetry-proto).
+
+The receiver will watch the directory and read files. If a file is updated or added,
+the receiver will read it in its entirety again.
+
+Please note that there is no guarantee that exact field names will remain stable.
+This intended for primarily for debugging Collector without setting up backends.
+
+Supported pipeline types: traces, metrics, logs
+
+## Getting Started
+
+The following settings are required:
+
+- `path` (no default): folder to watch.
+
+Example:
+
+```yaml
+receivers:
+  file:
```

Comment:
With this being a very limited receiver I'm not sure that it is appropriate to use the very general component type identifier `file`.  Something more specific, reflecting its limited scope, such as `otlpjsonfile` might be more appropriate.

Reply:
This matches the `file` exporter.

Reply:
Two poorly named components is worse than one.  Let's not make things worse.

Reply:
OK, but I have no imagination. What would you change the name to?

Reply:
Oh my bad. I see your suggestion above. Sold!